bench/wtperf/wtperf.c

```diff
@@ -2691,6 +2677,12 @@ main(int argc, char *argv[])
     /* If creating, remove and re-create the home and tiered bucket directories. */
     if (opts->create != 0) {
         testutil_recreate_dir(wtperf->home);
+        /* If running backups move down a directory level. */
+        if (opts->backup_interval != 0 && opts->backup_complete != 0) {
+            if (chdir(wtperf->home) != 0)
+                testutil_die(errno, "parent chdir: %s", wtperf->home);
```

Comment:
Why is it "parent" chdir? To me, it implies we are going back to the parent directory, but I don't think that's what we're doing here?

Reply:
Good catch. I copied it from timestamp abort and didn't change that. Fixed.